src/components/Button/Button.module.css

```diff
@@ -1,8 +1,7 @@
 .button {
   border-radius: var(--border-radius-small);
-  line-height: inherit;
```

Comment:
Is there a reason for this being removed?

Reply:
line-height is set by the css reset on that same element. If I left this line here, the line-height from the css reset will be overwritten by this line.